docs/linux.md

```diff
@@ -10,6 +10,14 @@ Install Ollama running this one-liner:
 curl -fsSL https://ollama.com/install.sh | sh
 ```
 
+### Custom install location
```

Comment:
Would it be possible instead to show how to curl the tar file and extract it in place (similar to the Go install experience) for `Custom install` ?